lib/models/algo_order.js

```diff
@@ -9,9 +9,7 @@ module.exports = () => ({
     algoID: String,
     state: Object,
     active: Boolean,
-
     exchangeData: Object
   },
-
-  mapKey: ({ gid, algoID }) => (`${gid}-${algoID}`)
+  mapKey: ({ gid }) => gid
```

Comment:
will have to be careful if we update package in HF application ..

Reply:
on slack:

> Abhishek Shrestha  2:03 AM
for HF app, we are resuming/cancelling old algo orders when HF app is opened ..  if a new version of HF is installed and suppose it uses this change .. it wont fetch the previously paused algo orders or cant really update it as well

>Tarcisio Ferraz  10:01 AM
Makes a lot of sense
10:03
since I won't use that function in the mongo adapter and the new ids are compatible (long numeric) I'll keep the original mapKey